claudio-plugin/skills/gws-doc-action-extractor/tests/test_arg_validation.py

```
@@ -0,0 +1,108 @@
+import subprocess
+
+import pytest
+
+_SCRIPTS = pathlib.Path(__file__).parent.parent / "scripts"
+EXTRACT_ACTIONS = _SCRIPTS / "extract_actions.sh"
+EXTRACT_RECENT = _SCRIPTS / "extract_recent_actions.sh"
+SCAN_DOCS = _SCRIPTS / "scan_docs.sh"
+
+
+def run(script, *args):
+    return subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)